src/main/scala/cognite/spark/v1/PushdownUtilities.scala

```diff
@@ -105,23 +105,30 @@ object PushdownUtilities {
   // PrunedFilteredScan, hence it's ok that our pushdown filter reads some data that should ideally
   // be filtered out
   // scalastyle:off
-  def getFilter(filter: Filter): PushdownExpression =
+  def getFilter(filter: Filter): PushdownExpression = {
+    def toStr(v: Any): String =
+      v match {
+        // let's hope that null-byte is invalid anywhere else
+        case i: Iterable[Any] => i.mkString("\u0000")
```

Comment:
Could do with a better explanation and perhaps an example. The null should be made into a `val` and reused in `idsFromWrappedArray` so the connection is made more clear.